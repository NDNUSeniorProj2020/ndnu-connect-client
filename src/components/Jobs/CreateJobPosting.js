import React, { useState } from 'react';
import { Input, Form, Radio } from 'antd';

export default function CreateJobPosting() {
	const { TextArea } = Input;
	const [title, setTitle] = useState('');
	const [company, setCompany] = useState('');
	const [location, setLocation] = useState('');
	const [description, setDescription] = useState('');
	const [type, setType] = useState('');
	
	return (
		<div>
			<Form>
				<Form.Item label="Title" name="title">
					<Input value={title} onChange={e => setTitle(e.target.value)} placeholder="Enter job title." />
				</Form.Item >
				<Form.Item label="Company Name" name="company">
					<Input value={company} onChange={e => setCompany(e.target.value)} palceholder="Enter company name." />
				</Form.Item>
				<Form.Item label="Loaction" name="location">
					<Input value={location} onChange={e => setLocation(e.target.value)} palceholder="Enter location." />
				</Form.Item>
				<Form.Item label="Description" name="description">
					<TextArea value={description} onChange={e => setDescription(e.target.value)} rows={8} />
				</Form.Item>
				<Form.Item label="Type" name="type">
					<Radio.group onChange={e => setType(e.target.value)} value={type}>
						<Radio value="FULL">Full-Time</Radio>
						<Radio value="PART">Part-Time</Radio>
						<Radio value="INTR">Internship</Radio>
					</Radio.group>
				</Form.Item>
			</Form>
		</div>
	);
}
