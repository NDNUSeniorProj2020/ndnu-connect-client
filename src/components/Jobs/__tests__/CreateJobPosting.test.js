import React from 'react';
import { shallow } from 'enzyme';

import CreateJobPosting from '../CreateJobPosting';

describe('tests for CreateJobPosting', () => {
	describe('snapshot tests', () => {
		it('renders without crashing', () => {
			const tree = shallow(<CreateJobPosting />);
			expect(tree).toMatchSnapshot();
		});
	});
});
