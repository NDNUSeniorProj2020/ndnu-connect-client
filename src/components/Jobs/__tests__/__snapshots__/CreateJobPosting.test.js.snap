// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for CreateJobPosting snapshot tests renders without crashing 1`] = `ShallowWrapper {}`;
